src/main/java/frc/robot/commands/manipulator/commandgroup/ShooterSequence.java

```diff
@@ -0,0 +1,18 @@
+package frc.robot.commands.manipulator.commandgroup;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants.ManipulatorConstants;
+import frc.robot.commands.manipulator.pivot.PivotPID;
+import frc.robot.commands.manipulator.shooter.Feeder;
+import frc.robot.commands.vision.AlignScore;
+
+public class ShooterSequence extends SequentialCommandGroup{
+    
+    public ShooterSequence(double shootSpeed, double pivotGoal) {
+    addCommands(
+        new AlignScore(), 
```

Comment:
We need a better way to implement this, this command will never end, thus we never reach the parallel command group to spin up the wheels and to up the pivot.

Reply:
I kind of put this here as a place holder honestly. Once we touch up align score we can add it to the shooter sequence.